README.md

```diff
@@ -37,21 +36,97 @@ Deployment of NVA consists of 2 templates which is used to create resources in 2
   | OrcidClientSecret | Secret to authenticate for ORCID | {{secretsmanager:OrcidClientSecret}} |
   | AlmaSruHost | URL to ALMA Library services | {{secretsmanager:AlmaSruHost}} |
   
-  ---
-**Current modules in NVA API and their deployment**
-
-|API path|GIT repository|SAR App Name|Version|Notes|
-|-----|-----|-----|-----|-----|
-|/alma|nva-alma-proxy|SruLastPublication|0.1.2| |
-|/upload|nva-upload-multipart|UploadMultipart|0.1.6| |
-|/person|nva-bare-proxy|PersonData|0.1.3| |
-|/publication|nva-publication-api|NvaPublicationApi|0.1.11| |
-|/channel|nva-channel-registry|PublicationChannelRegister|0.1.2| |
-|/doi-fetch|nva-fetch-doi|nva-fetch-doi|0.1.7| |
-|/doi-requests|nva-doi-requests-api|NvaDoiRequestsApi|0.1.0| |
-|/project|nva-cristin-projects|Projects|0.1.3| |
-|/institution|nva-institution-proxy|NvaInstitutionProxy|0.1.3| |
-|/download|nva-download-file|NvaDownloadPublicationFileApi|0.1.4| |
-|/customer|nva-customer-api|NvaCustomerApi|0.1.3|2020-09-09| |
-|/users-roles|nva-user-access-service|NvaUsersAndRolesService|0.1.2|2020-09-09| |
-| |nva-cognito-post-authentication-trigger|NvaCognitoPostAuthenticationTrigger|0.1.1|2020-09-09| |
+***
+
+## Deployment steps
+
+ 1. Identify repositories with changes.
+ 2. Merge develop into master for each repository with changes.
+ 3. Create new Github release in repository.
+ 4. Verify CodePipeline for publishing to SAR (Serverless Repo) has run successfully by checking versions and and date in SAR.
+ 5. Update deployment templates with new applications, versions and paramters (if any), see above.
+ 6. Verify that any external resources have been updated (DynamoDB, Secrets, Elasticsearch, Cognito etc).
+ 7. Deploy templates by updating stacks in Cloudformation (from local file but ensure that branch is up to date).
+ 8. Verify changes to stack is Completed.
+ 9. Deployment is done.
+
+## NVA Applications Overview
+
+Overview of Serverless Applications in the NVA platform.
+
+### Frontend
+
+**[Frontend](https://github.com/BIBSYSDEV/NVA-frontend)**
+
+Frontend for the NVA application.
+
+### APIs
+
+These applications use DynamoDB tables. Ensure all tables and indices are up to date before updating
```

Comment:
Since at the moment the Dynamo tables are not in the repositories, should we add a link here to the repo containing the Dynamo tables? If of course all Dynamo tables are in the same repo,

Reply:
Do we have Dynamo tables that are not residing together with the service?

I prefer each micro service to own its own data, micro services only solve an organization problem where multiple developers want to work on stuff at the same time. So then it is important they own the data they work with. 

So if we have Dynamo tables created from somewhere else, they should belong somewhere I hope? @axthosarouris

Reply:
After a recent update to nva-publication-api I think they all are located in the repo with the service using it Except for nva-doi-requests-api, but that should be subject to change.